ClangPowerTools/ClangPowerTools/Helpers/ReflectionManager.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Reflection;
+
+namespace ClangPowerTools.Helpers
+{
+  public static class ReflectionManager
```

Comment:
About the naming of the class `ReflectionManager` sounds very unrelated to what this class does. Maybe something like `GetSetProperty` is more intuitive. Of course, by specifying the purpose of the class in its name we will rename the class methods in `Get` and `Set`. What do you think?

Reply:
Yes, you're right! I'll do that, thank you for your remarks.